tools/create_app.py

```diff
@@ -1,283 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
Could you restore this file along with the modifications you've done to it in our `mac-application` repo? And could you move it to `installers/mac_installer.py`?

The thing is we decided with @dalthviz to create the Windows installer using a Github workflow on this repo. So we can do the same for the macOS one.

Reply:
Also, please move the bash script in `mac-application` to `installers` too.

Reply:
Does that mean we'll get rid of the mac application repo?
I'm good with that. That was my original intent anyway: have the application build part of the release workflow of this repo (GitHub actions).

However, if you don't mind, I'd like to migrate that stuff over from the mac-application repo as a separate PR after we get these initial PRs resolved. Then I know that all is working as it should and we can work on migrating stuff over and adding the GitHub actions.

Reply:
> Does that mean we'll get rid of the mac application repo?

Yeah, it does.

> I'm good with that. That was my original intent anyway: have the application build part of the release workflow of this repo (GitHub actions).

Right, @dalthviz came to same conclusion. Sorry if I made you waste your time with that (I thought it was better to have separate repos for the Windows and macOS installers, but you proved me wrong).

> However, if you don't mind, I'd like to migrate that stuff over from the mac-application repo as a separate PR after we get these initial PRs resolved.

Sure, no problem with that. Let's merge this one then.